data/yaml/split_joint.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
```

Comment:
These files are all in `data` -- can you move them to (a subdirectory of) `bin`? I know they are not for everyday use, but I would be surprised to find them in the data dir.

Reply:
They're really just one-time use and I plan to remove them once the merge is complete.